packages/compass-components/src/components/item-action-controls.tsx

```diff
@@ -269,6 +269,7 @@ export function ItemActionGroup<Action extends string>({
         if (tooltip) {
           return (
             <Tooltip
+              key={action}
```

Comment:
I guess this was already true before, but the file should probably have `export type ItemAction<Action extends string> = {` above instead of just `export type ItemAction<Action> = {` since we’re expecting `action` to be a string, not just any type of value

Reply:
Sure, updated in https://github.com/mongodb-js/compass/pull/4565/commits/37eb72ddd7fbeacc6008dd240c46c0e0f213197e